data/campaigns/Heir_To_The_Throne/units/Dark_Queen.cfg

```diff
@@ -60,9 +64,15 @@
             name=scepter
         [/filter_attack]
         start_time=-200
+        swoosh_start_time=-50
         [frame]
-            image="units/human-queen-staff-[1~4].png:[100*3,75]"
+            image="units/human-queen.png:100,units/human-queen-attack.png:200,units/human-queen.png:1"
         [/frame]
+        [swoosh_frame]
+            image="units/human-queen-swoosh.png:100,misc/blank-hex.png:1"
+	    auto_vflip=no
```

Comment:
Arguably, he shouldn't switch from the default Orcish Archer portrait to the unit image. Yet if the portraits stays the same through the transformation, it's kind of unclear what the whole schtick is about. Some kind of frankenstein portrait combining his first portrait (which can be something other than Orcish Archer, if it makes frankensteining easier) with some merfolk equipment seems like the best route, if one can find an even remotely natural-looking combination. I can look into it and see if I can find something that could work.